### ISSUE TITLE
FEATURE: LMS Hook for Track & State Updates

### ISSUE THREAD

**Issue by mtudor** [mtudor/HomeySqueezebox#3]
Such that the LMS will ping us when the track changes, volume changes, play stops / stars etc. and this can be updated in the Homey interface.

**Comment by dvanp** [comment]
Would love this feature, but is this app still being developed?

**Comment by Inversion-NL** [comment]
Currently not very, no.
We're planning on continuing later on

**Comment by mtudor** [comment]
Yep, that's pretty much it. We're both still here but time is short at the moment. Rest assured though that we want our Squeezeboxes singing and dancing with Homey too ;) Are you a potential contributor @dvanp? No worries if not, we'll keep you informed!

_Sent from my Samsung SM-G955F using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by dvanp** [comment]
I might be able to contribute, but unfortunately haven't discovered yet how to add these features. If somebody can give me some pointers I will try again!

**Comment by dvanp** [comment]
My main problem is with figuring out how to get Homey notified of state/song changes. I've got getCurrentTitle showing up on the mobile card, but it is only updated on init, so when the app is started.

**Comment by Inversion-NL** [comment]
You'll have to emit an real-time event:
https://apps.developer.athom.com/v1/library/drivers/index.html

module.exports.realtime( device_data, capability, newValue );
